iOS-Email-Client/Model/EventData.swift

```diff
@@ -19,6 +19,7 @@ class EventData {
         var opens = [FeedItem]()
         var modifiedThreadIds = [String]()
         var modifiedEmailKeys = [Int]()
+        var fromWS = false
```

Comment:
what's this?

Reply:
if the event response comes from the websocket, i do not ack the event nor update the loading flag in mailbox